src/main/java/com/codeborne/selenide/Credentials.java

```diff
@@ -0,0 +1,34 @@
+package com.codeborne.selenide;
+
+import java.util.Base64;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+public final class Credentials {
```

Comment:
I would move this class to package `com.codeborne.selenide.impl` and make package-private. It's only used in `Navigator`.

Reply:
@asolntsev 
I consider to move this class to top level because I think about extra `open` method with signatures containing `credentials` instead of `login` and `password` pair.
What do you thinks?

Reply:
Yes, one parameter `Credentials` is better than two Strings "login"/"password".

Reply:
Or we can just overload method and give ability to do open with `Credentials` and `login/password` pair?